src/main/java/com/joona/config/LibraryApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.joona.config;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan("com.joona")
+public class LibraryApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(LibraryApplication.class,args);
+    }
```

Comment:
Add log (time being you can use system:: out) to indicate application came up successfully